test/python/test_onnxruntime_genai_e2e.py

```diff
@@ -53,5 +53,7 @@ def run_model(model_path: str | bytes | os.PathLike):
     for model_name in ["microsoft/phi-2"]:
         with tempfile.TemporaryDirectory() as temp_dir:
             device = "cuda" if og.is_cuda_available() else "cpu"
-            download_model(temp_dir, device, model_name)
+            download_model(temp_dir, device, model_name, int4)
             run_model(temp_dir)
+            download_model(temp_dir, "cpu", model_name, fp32)
```

Comment:
Do you not want to execute the model in this test?

Reply:
Fixed

Reply:
I think you can do this:

```python
    for precision in ["int4", "fp32"]:
        with tempfile.TemporaryDirectory() as temp_dir:
            device = "cuda" if og.is_cuda_available() else "cpu"
            download_model(temp_dir, device, model_name, precision)
            run_model(temp_dir)
```

Reply:
Updated